src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/ActionButtons.tsx

```diff
@@ -62,25 +81,39 @@ const ActionButtons = ({
 
   return (
     <div className='discovery-modal_buttons-row' data-testid='actionButtons'>
-      <DownloadAllModal
+      <DownloadModal
         downloadStatus={downloadStatus}
         setDownloadStatus={setDownloadStatus}
       />
       <Row className='row'>
-        {/*
-        <Col flex='1 0 auto'>
-          <Button className='discovery-action-bar-button'>
-            Download <br />
-            Variable-Level Metadata
-          </Button>
-        </Col>
-        */}
-        {showDownloadStudyLevelMetadataButtons && (
+        {showDownloadVariableMetadataButton && (
```

Comment:
when user has not logged in, **and if this study has a VLMD**, this button should show as `Login to To Download Variable-Level Metadata`, and redirect to user to the login page upon clicking

Reply:
Per discussion with stakeholders Feb 22, will not implement this functionality—users will be able to download study metadata whether they are logged in or not.